### ISSUE TITLE
Error running server

### ISSUE THREAD

**Issue by juancarloslin** [spoman007/face-login-website#1]
Try to test app, 

register ok, 2 pictures created under 

and when try to login, followed error
[lin@localhost face-login-website]$ cd labeled_images/
[lin@localhost labeled_images]$ ls
Lin  Thor
[lin@localhost labeled_images]$ ls Lin
1.jpg  2.jpg
[lin@localhost labeled_images]$ sudo npm start
[sudo] password for lin: 

> node_starter@1.0.0 start /home/lin/Downloads/node/face-login-website
> babel-watch --exec babel-node src/index.js

cpu backend was already registered. Reusing existing backend factory.
Platform node has already been set. Overwriting the platform with [object Object].
2020-05-29 21:44:44.479178: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2893520000 Hz
2020-05-29 21:44:44.479551: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x642ae70 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-29 21:44:44.479587: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
Server listening on port 5000
(node:22322) UnhandledPromiseRejectionWarning: Error: 'softmax' not yet implemented or not found in the registry. Did you forget to import the kernel?
    at notYetImplemented (/home/lin/Downloads/node/face-login-website/node_modules/@tensorflow/tfjs-core/src/backends/backend.ts:665:9)
    at NodeJSKernelBackend.KernelBackend.softmax (/home/lin/Downloads/node/face-login-website/node_modules/@tensorflow/tfjs-core/src/backends/backend.ts:309:12)
    at engine_1.ENGINE.runKernelFunc.logits (/home/lin/Downloads/node/face-login-website/node_modules/face-api.js/node_modules/@tensorflow/tfjs-core/src/ops/softmax.ts:64:27)
    at /home/lin/Downloads/node/face-login-website/node_modules/face-api.js/node_modules/@tensorflow/tfjs-core/src/engine.ts:586:31
    at /home/lin/Downloads/node/face-login-website/node_modules/face-api.js/node_modules/@tensorflow/tfjs-core/src/engine.ts:424:20
    at Engine.scopedRun (/home/lin/Downloads/node/face-login-website/node_modules/face-api.js/node_modules/@tensorflow/tfjs-core/src/engine.ts:435:19)
    at Engine.tidy (/home/lin/Downloads/node/face-login-website/node_modules/face-api.js/node_modules/@tensorflow/tfjs-core/src/engine.ts:422:17)
    at kernelFunc (/home/lin/Downloads/node/face-login-website/node_modules/face-api.js/node_modules/@tensorflow/tfjs-core/src/engine.ts:586:20)
    at /home/lin/Downloads/node/face-login-website/node_modules/face-api.js/node_modules/@tensorflow/tfjs-core/src/engine.ts:599:23
    at Engine.scopedRun (/home/lin/Downloads/node/face-login-website/node_modules/face-api.js/node_modules/@tensorflow/tfjs-core/src/engine.ts:435:19)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:22322) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:22322) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:22322) UnhandledPromiseRejectionWarning: Error: 'softmax' not yet implemented or not found in the registry. Did you forget to import the kernel?
    at notYetImplemented (/home/lin/Downloads/node/face-login-website/node_modules/@tensorflow/tfjs-core/src/backends/backend.ts:665:9)
    at NodeJSKernelBackend.KernelBackend.softmax (/home/lin/Downloads/node/face-login-website/node_modules/@tensorflow/tfjs-core/src/backends/backend.ts:309:12)
    at engine_1.ENGINE.runKernelFunc.logits (/home/lin/Downloads/node/face-login-website/node_modules/face-api.js/node_modules/@tensorflow/tfjs-core/src/ops/softmax.ts:64:27)
    at /home/lin/Downloads/node/face-login-website/node_modules/face-api.js/node_modules/@tensorflow/tfjs-core/src/engine.ts:586:31
    at /home/lin/Downloads/node/face-login-website/node_modules/face-api.js/node_modules/@tensorflow/tfjs-core/src/engine.ts:424:20
    at Engine.scopedRun (/home/lin/Downloads/node/face-login-website/node_modules/face-api.js/node_modules/@tensorflow/tfjs-core/src/engine.ts:435:19)
    at Engine.tidy (/home/lin/Downloads/node/face-login-website/node_modules/face-api.js/node_modules/@tensorflow/tfjs-core/src/engine.ts:422:17)
    at kernelFunc (/home/lin/Downloads/node/face-login-website/node_modules/face-api.js/node_modules/@tensorflow/tfjs-core/src/engine.ts:586:20)
    at /home/lin/Downloads/node/face-login-website/node_modules/face-api.js/node_modules/@tensorflow/tfjs-core/src/engine.ts:599:23
    at Engine.scopedRun (/home/lin/Downloads/node/face-login-website/node_modules/face-api.js/node_modules/@tensorflow/tfjs-core/src/engine.ts:435:19)
(node:22322) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)

[lin@localhost labeled_images]$ npm -v
6.14.5
[lin@localhost labeled_images]$ node -v
v14.2.0
[lin@localhost labeled_images]$ cat /etc/redhat-release 
Fedora release 31 (Thirty One)
[lin@localhost labeled_images]$ uname -a
Linux localhost.localdomain 5.5.17-200.fc31.x86_64 #1 SMP Mon Apr 13 15:29:42 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

**Comment by spoman007** [comment]
@juancarloslin There is a good chance that this issue is because of version differences.
I have checked in the package-lock.json in repository.
Can you remove your node_modules and package-lock.json then take latest and try installing dependencies.
Hope this helps. Let me know.
